HISTORY.rst

```diff
@@ -1,6 +1,15 @@
 History
 =======
 
+Unreleased
+----------
+
+- Cassette names sanitation is now more robust and replaces more special chars;
```

Comment:
Let's reword this as:

```
Previous attempts to sanitize cassette names were incomplete. Sanitization has become more thorough which could have some affects on existing cassette files. **This may cause new cassettes to be generated.**
```

Reply:
changed. amended.